src/server/api/controllers/waypoints.controller.js

```diff
@@ -0,0 +1,66 @@
+const HttpError = require("../lib/utils/http-error");
+const knex = require("../../config/db");
+
+// Method for getting waypoints by id:
+const getWaypointsById = async id => {
+  try {
+    const waypoints = await knex("waypoints")
+      .select("id", "lon", "lat", "created_at", "updated_at")
```

Comment:
I know your PR was first in the line, but I needed to get the DB changes in there first, so unfortunately these have changed names to "longitude" and "latitude" respectively.

Reply:
Make sure to check the rest of the PR for the same changes.